api/app/Aggregates/Events/EventAggregate.php

```diff
@@ -33,7 +33,7 @@ public function toArray(): array
             'id' => $this->id,
             'invitee_email' => $this->inviteeEmail,
             'start_date' => $this->startDate,
-            'event_type' => $this->eventType,
+            'event_type' => $this->eventTypeId,
```

Comment:
Please remove the changes in the file to avoid merge conflicts. It is done in another PR.